app/alerts.py

```diff
@@ -72,6 +76,24 @@ def build_departments_geojson():
     return geojson
 
 
+# ----------------------------------------------------------------------------------------------------------------------
+# Site devices
+# The following block is used to load the data relative to site devices.
+# It is called in main.py by the callback that is itself triggered by the user's login.
+
+def get_site_devices_data(client):
```

Comment:
Dumb comment: maybe we could remove the blank lines ?

Reply:
Done, too!